src/frontend/src/optimizer/plan_node/logical_filter.rs

```diff
@@ -185,11 +190,120 @@ impl ToBatch for LogicalFilter {
     }
 }
 
+fn convert_comparator_to_priority(comparator: Type) -> i32 {
+    match comparator {
+        Type::Equal => 0,
+        Type::GreaterThan | Type::GreaterThanOrEqual => 1,
+        Type::LessThan | Type::LessThanOrEqual => 2,
+        _ => -1,
+    }
+}
+
 impl ToStream for LogicalFilter {
     fn to_stream(&self) -> Result<PlanRef> {
         let new_input = self.input().to_stream()?;
-        let new_logical = self.clone_with_input(new_input);
-        Ok(StreamFilter::new(new_logical).into())
+
```

Comment:
Generally, can we add more comments explaining the some of the code blocks? It will make it easier for someone to read.

Reply:
> Generally, can we add more comments explaining the some of the code blocks? It will make it easier for someone to read.

fixed

Reply:
thanks